deployment/live/example-gcp/terragrunt.hcl

```diff
@@ -3,9 +3,9 @@ terraform {
 }
 
 locals {
-  project_id = "trillian-tessera"
-  location   = "us-central1"
-  base_name   = "example-gcp"
+  project_id = get_env("GOOGLE_PROJECT", "trillian-example")
```

Comment:
Is this supposed to be `trillian-tessera` to match the old version? I think this name is particularly confusing given we have a repo with (almost) this name which is for the old trillian!

Reply:
Duh. Done, thanks.